pkg/sql/set_zone_config.go

```diff
@@ -183,6 +183,14 @@ func (p *planner) SetZoneConfig(ctx context.Context, n *tree.SetZoneConfig) (pla
 		return nil, err
 	}
 
+	if !p.ExecCfg().Codec.ForSystemTenant() &&
+		!secondaryTenantZoneConfigsEnabled.Get(&p.ExecCfg().Settings.SV) {
+		return nil, pgerror.Newf(pgcode.FeatureNotSupported,
+			"secondary tenants cannot set zone configurations unless %s is enabled",
```

Comment:
Since this is an experimental cluster setting we want to hide from users, it's fine to not refer to the setting name. "zone configs disabled for multi-tenancy".

Reply:
Good point, I'll just re-use the "unsupported with multi-tenancy" here.